dokku-deploy/README.md

```diff
@@ -0,0 +1,77 @@
+# dokku-deploy
+
+This action makes deployment to Dokku as easy as possible!
+Based on [idoberko2/dokku-deploy-github-action](https://github.com/idoberko2/dokku-deploy-github-action).
+
+## Inputs
+
+### ssh-private-key
```

Comment:
I think it is better to specify that is a required field here.

```
ssh-private-key (required)
```

Reply:
done